product_supplierinfo_for_customer_sale_sol_code_hide/static/description/index.html

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ouch, this line again!! It's weird, since you've update OCA maintainer tools this line shouldn't be added anymore, please check